README.md

```diff
@@ -1,6 +1,7 @@
 # IMPORTANT NOTE
-This repository will be removed on December 14th and will no longer be maintained and ownership will also not be transferred. Please fork this repositiory if you wish to continue to use it.
-
+Since the base repository is archived, this fork exists only to ensure that Git
+LFS has a copy of this code to depend on. The Git LFS project doesn't actively
+maintain this repository and therefore we can't promise to fix bugs here.
```

Comment:
I think it would be worth being more explicit about the fact that we won't be doing any development or maintenance of this repository. Maybe something like

```
Since the base repository is archived, this fork exists only to ensure that Git
LFS has a copy of this code to depend on. The Git LFS project doesn't actively
maintain this repository, and therefore we won't be adding new features or fixing any bugs.
```

Reply:
I think that your suggestion is good, though I prefer @bk2204's wording. Of course, if there is a security issue, we should leave open the door that we'd fix it. Other than that, I think the wording is clear in context that we don't really anticipate to be working with this much.

Reply:
@ttaylorr I think that's a good point with regards to security issues. In that case, i'm happy with using the original wording